Scripts/DCS-BIOS/lib/modules/Module.lua

```diff
@@ -1011,6 +1012,14 @@ function Module.round(num)
 	return num >= 0 and math.floor(num + 0.5) or math.ceil(num - 0.5)
 end
 
+--- rounds a number to certain decimal place
+--- @param num number the number to round
+--- @param decimal_places integer
+--- @return number
+function Module.round2(num, decimal_places)
+	return tonumber(string.format("%." .. (decimal_places or 0) .. "f", num)) or 0
```

Comment:
I haven't looked into it, but there must be a more efficient way to round than converting a value to a string and then back to a number again?

I suspect this is also a losing battle since some floats ([like 0.3](https://0.30000000000000004.com/)) basically can't be rounded

Reply:
Googling the subject it seems that lua has weak support for rounding. This was suggested as the common way to do it on Stackoverflow.

Reply:
https://stackoverflow.com/questions/18313171/lua-rounding-numbers-and-then-truncate

Reply:
It's common, but not necessarily performant. At any rate, I suspect that my PR #392 should solve the problem at hand (and it's in the core set value code so it will apply to all controls).